src/modules/commander/commander.cpp

```diff
@@ -2576,8 +2576,8 @@ set_control_mode()
 		control_mode.flag_control_attitude_enabled = true;
 		control_mode.flag_control_altitude_enabled = true;
 		control_mode.flag_control_climb_rate_enabled = true;
-		control_mode.flag_control_position_enabled = true;
-		control_mode.flag_control_velocity_enabled = true;
+		control_mode.flag_control_position_enabled = !vtol_status.vtol_in_trans_mode;
```

Comment:
If the struct is memset with 0 every boolean should be false right? I changed it to use the vehicle status instead of vtol status because these flags only get updated in vtol configuration (but the initialization is the same for that, memset 0).